src/commands/ImportObjectsCommandGenerator.js

```diff
@@ -295,10 +280,22 @@ module.exports = class ListObjectsCommandGenerator extends BaseCommandGenerator
 			throw TranslationService.getMessage(MESSAGES.CANCEL_IMPORT);
 		}
 
+		const flags = [];
+		if (
+			this._projectInfoService.getProjectType() === PROJECT_ACP &&
+			(answers[COMMAND_FLAGS.EXCLUDE_FILES] ||
```

Comment:
This check is to handle non-interactive mode wherein the user may or may not include the -excludefiles flag.

We delete them from answers because they are no longer needed after their value has been read and taken into account when the -excludefiles flag has been added to to the list of flags.

Reply:
I think this is not explicit enough and will lead to confusion.
If we need different behavior based on wether the CLI is running in interactive vs non-interactive, then we need to make that explicit.

We need a small refactor to send to the command the mode (interactive vs non-interactive).
In CommandInstanceFactory, we need to add:
```
	const generatorInstance = new Generator({
			commandMetadata,
			projectFolder: options.projectFolder,
			executionPath: options.executionPath,
			runInInteractiveMode: options.runInInteractiveMode,   //ADD THIS PARAM
		});
```
And then in the BaseCommandGenerator, add the following param to the constructor:
```
 this._runInInteractiveMode = options.runInInteractiveMode;
```

Then in this preExecuteAction, we can use:
```
if(this._runInInteractiveMode){
...
}
```
This will be much more explicit.

Reply:
Change made.